src/raps/actions/ens.ts

```diff
@@ -380,6 +442,35 @@ const ensAction = async (
           category: 'profiles',
         });
         break;
+      case ENSRegistrationTransactionType.SET_ADDR:
+        tx = await executeSetAddr(
+          name,
+          ensRegistrationRecords,
+          gasLimit,
+          maxFeePerGas,
+          maxPriorityFeePerGas,
+          wallet,
+          nonce
+        );
+        analytics.track('Edited ENS records', {
+          category: 'profiles',
+        });
+        break;
+      case ENSRegistrationTransactionType.RECLAIM:
+        tx = await executeReclaim(
+          name,
+          parameters.ownerAddress ?? ownerAddress,
```

Comment:
i think this is where it gets confusing, `parameters.ownerAddress` is the new address that is going to be the new owner of the ENS, `ownerAddress` is the `accountAddress` from settings.
They're different but they have the same name
https://github.com/rainbow-me/rainbow/pull/3389/files#r895787462